napari/_qt/_tests/test_app.py

```diff
@@ -24,7 +24,7 @@ def test_qapp(qapp):
     assert qapp == deprecated_qapp
 
 
-@pytest.mark.skipif(os.name != 'Windows', reason='Windows specific')
+@pytest.mark.skipif(os.name != 'nt', reason='Windows specific')
```

Comment:
😮 

(confirmed in [the docs](https://docs.python.org/3/library/os.html#os.name). But crazy to have such a basic error in there. 😅)